check.go

```diff
@@ -443,9 +447,9 @@ func (c *CheckRunner) reapServicesInternal() {
 				Node:      check.Node,
 				ServiceID: serviceID,
 			}, nil)
-			c.logger.Printf("[INFO] agent: Check %q for service %q has been critical for too long (%s | timeout: %s); deregistered service",
+			c.logger.Info("agent: ", "", hclog.Fmt("Check %q for service %q has been critical for too long (%s | timeout: %s); deregistered service",
 				checkID, serviceID,
-				time.Since(criticalTime), timeout)
+				time.Since(criticalTime), timeout))
```

Comment:
Looks like there is an extra `"", ` in the second parameter of Info() that isn't used. This could also be reorganized to something like:
```
c.logger.Info("Agent: check for service has been critical for too. long", "checkId", checkID, "serviceId", serviceID, "timeSince", time.Since(criticalTime), "timeout", timeout)
```

Reply:
Not sure if you missed the above comment or you disagree. Please let me know!